config/schema.yaml

```diff
@@ -694,6 +655,66 @@ schema:
         type: string
         doc: A string value composed only of letters, numbers, and underscores.
 
+  dataUnitRegions:
+    -
+      relates:
+        - Visit
+        - Sensor
+      tables:
+        VisitSensorRegion:
+          doc: >
+            A one-to-many table join table that stores the spatial region
+            associated with a particular Visit-Sensor combination. Entries are
+            expected to be calculated outside the database and added/updated
+            whenever the Visit's region is.  This table contains exactly one
+            entry for each Visit+Sensor combination.
+          columns:
+          -
+            name: camera
+            type: string
+            primary_key: true
+            nullable: false
+            doc: Name of the Camera associated with the Visit and Sensor.
+          -
+            name: visit
+            type: int
+            primary_key: true
+            nullable: false
+            doc: Visit ID
+          -
+            name: sensor
+            type: int
+            primary_key: true
+            nullable: false
+            doc: visit
+          -
+            name: region
+            type: string
```

Comment:
I've made it a blob here too.  Is there a way to use magic Python conversions in a way that is truly RDBMS-independent?  (Note I'd like to avoid even exposing that we're using SQLAlchemy publicly).

Reply:
I don't think we need magic here, we are reading this schema ourselves and transform it into backed-specific schema so we have complete freedom. My worry is that using low-level (physical) type here instead of logical (`region`) we limit ourselves to a particular choice of storage type. I can imagine for some backends BLOB may be too inefficient and you may want to store regions (which should be relatively short strings of bytes) in more efficient storage type.

Reply:
Ah, I just misunderstood what you were suggesting.  I'll go ahead and make that change (while still mapping `region` columns to `sqlalchemy.LargeBinary` for now).